src/core_modules/capture-core/components/WidgetEnrollmentEventNew/FinishButtons/finishButtons.types.ts

```diff
@@ -0,0 +1,20 @@
+import type { ReactElement } from 'react';
+import type { AddEventSaveType } from '../DataEntry/addEventSaveTypes';
+
+export type InputProps = {
+    onSave: (saveType: AddEventSaveType) => void;
+    onCancel: () => void;
+    isLoading?: boolean;
+    cancelButtonIsDisabled?: boolean;
+    id: string;
+};
+
+export type Props = {
+    onSave: (saveType: AddEventSaveType) => void;
+    cancelButton: ReactElement;
+    isLoading?: boolean;
+    classes: {
+        container: string;
+        button: string;
+    };
```

Comment:
Remove `classes` and use `Props & WithStyles<typeof styles>` in src/core_modules/capture-core/components/WidgetEnrollmentEventNew/FinishButtons/FinishButtons.component.tsx line 24